### PR TITLE
py-cmseq: new package

### PR DESCRIPTION
**Pull request by luke-dt**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-cmseq/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-cmseq/package.py]:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Copyright year needs to be corrected.

Confirmed the sha256.  

Will defer python-related feedback to @adamjstewart if he is interested in commenting.

File: var/spack/repos/builtin/packages/py-cmseq/package.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyCmseq(PythonPackage):
+    """CMSeq is a set of commands to provide an
+       interface to .bam files for coverage and sequence
+       consensus. Used as a dependency for py-metaphlan."""
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-cmseq/package.py]:
```suggestion
       consensus."""
```
This package could be a dependency of many other packages, no need to mention this.

File: var/spack/repos/builtin/packages/py-cmseq/package.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    homepage = "https://github.com/SegataLab/cmseq"
+    url      = "https://github.com/SegataLab/cmseq/archive/refs/tags/1.0.4.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-cmseq/package.py]: Let's instead download from PyPI, see https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

File: var/spack/repos/builtin/packages/py-cmseq/package.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    version('1.0.4', sha256='9d9412b0c58dfaef0d9e3621a0c4b7cd5330dbc1399370d3e69ba03959a26d68')
+
+    depends_on('py-numpy')
+    depends_on('samtools@1.2:')
+    depends_on('py-pysam')
+    depends_on('py-pandas')
+    depends_on('py-biopython@:1.76')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-cmseq/package.py]: Missing deps on scipy and bcbio-gff. These all need to be `type=('build', 'run')`

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Review by adamjstewart — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**